### ISSUE TITLE
[Suggestion] Sideloading Chex Quest 2 as a second episode for Chex Quest

### ISSUE THREAD

**Issue by RosalinaFan573** [fabiangreffrath/crispy-doom#1238]
I think the ability to sideload Chex Quest 2 as a second episode for Chex Quest in a similar fashion to NRFTL/Sigil would be nice. Perhaps it could even load custom episode name lumps if provided?

**Comment by fabiangreffrath** [comment]
I agree that would be nice. Though, I think I remember that Check Quest 2 is more than a mere "level pack" but also contains new sprites for some monsters and weapons?

**Comment by fabiangreffrath** [comment]
Just checked and yes indeed, chex2.wad contains a complete set of sounds, patches, flats and sprites. Not sure if these will work well together with Chex 1...

**Comment by fabiangreffrath** [comment]
I gave this a shot and got pretty far so that `chex2.wad` got auto-loaded if `chex.wad` is the IWAD and a second episode was added to the menu etc. However, the problem is that `chex2.wad` overrides *every* asset that `chex.wad` already contains. I would have to divert everything out of the way, music, sounds, sprites, flats, textures, *everything*. In fact, if you load an additional `PLAYPAL` and `COLORMAP` lump, then `chex2.wad` is a full-blown IWAD itself - I just tested.

So, sorry, nice idea but this isn't worth the effort!
